sys-auth/fprintd/fprintd-1.90.9.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson pam systemd
+
+DESCRIPTION="D-Bus service to access fingerprint readers"
+HOMEPAGE="https://gitlab.freedesktop.org/libfprint/fprintd"
+SRC_URI="https://gitlab.freedesktop.org/libfprint/${PN}/-/archive/v${PV}/${PN}-v${PV}.tar.bz2"
+
+LICENSE="GPL-2"
```

Comment:
Seems to be `GPL-2+`. At least the 1.90.8 is also gpl-2+, could you check 0.8 versions too, and update all relevant ebuilds with updated LICENSE field in a **separate** commit.

Reply:
@juippis I have requested stabilization for 1.90.8. Can I change the license without break the stabilization request?

Reply:
Yes, it doesn't affect runtime or list of installed files.

Reply:
Ok, I change them now